### PR TITLE
fix: ensure that `system.run()` lives inside a Task

### PR DESCRIPTION
**Pull request by aesedepece**
this is believed to help with #1088

### CODE REVIEW

File: node/src/actors/node.rs [hunk 1]
```diff
@@ -62,7 +62,9 @@ pub fn run(config: Config, callback: fn()) -> Result<(), failure::Error> {
     SystemRegistry::set(json_rpc_server_addr);
 
     // Run system
```
Comment (tmpolaczyk) [inline node/src/actors/node.rs:64]: Maybe add a short comment to the code explaining the change?

```suggestion
    // Run system inside a future
    // This is believed to help with issue #1088
```

Comment (aesedepece) [reply node/src/actors/node.rs]: Applied!

Comment (tmpolaczyk) [reply node/src/actors/node.rs]: The important line is the one that references the issue, and I think it got lost?

Comment (aesedepece) [reply node/src/actors/node.rs]: Don't worry, thanks for sending it once again. Updated!

**Pull request merged**